__tests__/app.test.js

```diff
@@ -15,3 +23,33 @@ describe('GET /api', () => {
       });
   });
 });
+
+describe('GET /api/topics', () => {
+  test('200: Responds with an array of topic objects containing slug and description properties', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
+          );
+        });
+      });
+  });
+});
+
+describe('Error handling: non-existent route queried', () => {
+  test('404: request to non-existent route', () => {
+    return request(app)
+    .get('/api/topisc')
```

Comment:
"topisc" is a likely version of what could be requested, but as our test is interested in anything that is not an endpoint (and humans are bad at spotting typos) could we put something a bit more descriptive / obvious here?